evergreen/eg_app/views.py

```diff
@@ -12,6 +18,54 @@
 def index(request):
     return render(request,'index.html')
 
+def getFileType(filePath):
+    return mimetypes.guess_type(filePath)[0]  
+
+def fileHandler(request, fileName): # can handle img and text 
+    # sanitizedfileName = quote(fileName) # get a clean full file path 
+    # path = Path(settings.STATIC_ROOT) / sanitizedfileName
+
+    path = fileName
+
+    allowedType = {'.css', '.html', '.js', '.png', '.jpg', '.jpeg', '.gif', '.mp3', '.mp4', '.xml', '.json', '.pdf','.ico'} # can add more, 
+
+    if not path.suffix.lower() in allowedType: # to deal with user uploads 
+        return HttpResponseNotFound("404 - File type not allowed")
+    
+    # #deal with /../ attacks 
+    # rootPath = Path(settings.STATIC_ROOT).resolve() #.resolve get absolute path 
+    # if not path.resolve().is_relative_to(rootPath):
+    #     return HttpResponseNotFound("404 - Invalid file path")
```

Comment:
Why is this added commented out? Same for sanitizedFilePath above

Reply:
sanitizedFilePath gets the absolute path
these lines deal with the user trys to access a different directory with a /../ attack